### ISSUE TITLE
Tests under "System.Security.Cryptography.Xml.Tests.SignedXmlTest" failed with "Xunit.Sdk.EqualException"

### ISSUE THREAD

**Issue by Jiayili1** [dotnet/corefx#19268]
Detail: https://ci.dot.net/job/dotnet_corefx/job/master/job/outerloop_netfx_windows_nt_debug/39/testReport/System.Security.Cryptography.Xml.Tests/SignedXmlTest/AsymmetricRSASignature/
https://ci.dot.net/job/dotnet_corefx/job/master/job/outerloop_netfx_windows_nt_debug/39/testReport/System.Security.Cryptography.Xml.Tests/SignedXmlTest/SignedXmlUsesSha256ByDefault/

Failed tests: 
 System.Security.Cryptography.Xml.Tests.SignedXmlTest.AsymmetricRSASignature 
 System.Security.Cryptography.Xml.Tests.SignedXmlTest.SignedXmlUsesSha256ByDefault

~~~
System.Security.Cryptography.Xml.Tests.SignedXmlTest.AsymmetricRSASignature [FAIL]
        Assert.Equal() Failure
                                          (pos 21)
        Expected: ···ttp://www.w3.org/2001/04/xmldsig-more#rsa-sha256
        Actual:   ···ttp://www.w3.org/2000/09/xmldsig#rsa-sha1
                                          (pos 21)
        Stack Trace:
          D:\j\workspace\outerloop_net---903ddde6\src\System.Security.Cryptography.Xml\tests\SignedXmlTest.cs(191,0): 
at System.Security.Cryptography.Xml.Tests.SignedXmlTest.AsymmetricRSASignature()

System.Security.Cryptography.Xml.Tests.SignedXmlTest.SignedXmlUsesSha256ByDefault [FAIL]
        Assert.Equal() Failure
                                          (pos 21)
        Expected: ···ttp://www.w3.org/2001/04/xmlenc#sha256
        Actual:   ···ttp://www.w3.org/2000/09/xmldsig#sha1
                                          (pos 21)
        Stack Trace:
          D:\j\workspace\outerloop_net---903ddde6\src\System.Security.Cryptography.Xml\tests\SignedXmlTest.cs(1579,0): 
at System.Security.Cryptography.Xml.Tests.SignedXmlTest.SignedXmlUsesSha256ByDefault()
~~~

**Comment by Jiayili1** [comment]
Following tests failed with same issue:
 System.Security.Cryptography.Xml.Tests.ReferenceTest.Ctor_Uri
 System.Security.Cryptography.Xml.Tests.ReferenceTest.Ctor_Stream

Detail: https://ci.dot.net/job/dotnet_corefx/job/master/job/outerloop_netfx_windows_nt_debug/39/testReport/System.Security.Cryptography.Xml.Tests/ReferenceTest/Ctor_Uri_uri__null_/

**Comment by karelz** [comment]
Fixed in #19376
